templates/config.edn

```diff
@@ -1,137 +1,160 @@
+ ;; Default value: true
+ :feature/enable-whiteboards? true
 
- ;; Disable the built-in Scheduled tasks and deadlines query
- ;; :feature/disable-scheduled-and-deadline-query? true
+ ;; Disable the journal's built-in 'Scheduled tasks and deadlines' query.
+ ;; Default value: false
+ :feature/disable-scheduled-and-deadline-query? false
 
- ;; Specify the number of days in the future to display in the
- ;; scheduled tasks and deadlines query, with a default value of 7 which
- ;; displays tasks for the next 7 days.
+ ;; Specify the number of days displayed in the future for
+ ;; the 'scheduled tasks and deadlines' query.
+ ;; Default value: 7
  ;; Example usage:
- ;; Display all scheduled and deadline blocks for the next 14 days
+ ;; Display all scheduled tasks and deadlines for the next 14 days:
  ;; :scheduled/future-days 14
+ :scheduled/future-days 7
 
- ;; Specify the date on which the week starts.
- ;; Goes from 0 to 6 (Monday to Sunday), default to 6
+ ;; Specify the first day of the week.
+ ;; Default value: 6 (Sunday)
+ ;; Available options:
+ ;;  - integer from 0 to 6 (Monday to Sunday)
  :start-of-week 6
 
- ;; Specify a custom CSS import
- ;; This option take precedence over your local `logseq/custom.css` file
- ;; You may find a list of awesome logseq themes here:
- ;; https://github.com/logseq/awesome-logseq#css-themes
- ;; Example:
- ;; :custom-css-url "@import url('https://cdn.jsdelivr.net/gh/dracula/logseq@master/custom.css');"
+ ;; Import a custom CSS file.
+ ;; This option takes precedence over the local `logseq/custom.css` file.
+ ;; :custom-css-url "@import url('');"
 
- ;; Specify a custom js import
- ;; This option take precedence over your local `logseq/custom.js` file
+ ;; Import a custom JS file.
+ ;; This option takes precedence over the local `logseq/custom.js` file.
  ;; :custom-js-url ""
 
  ;; Set a custom Arweave gateway
  ;; Default gateway: https://arweave.net
  ;; :arweave/gateway ""
 
  ;; Set Bullet indentation when exporting
- ;; default option: tab
- ;; Possible options for `:export/bullet-indentation` are
- ;; 1. `:eight-spaces` as eight spaces
- ;; 2. `:four-spaces` as four spaces
- ;; 3. `:two-spaces` as two spaces
- ;; :export/bullet-indentation :tab
-
- ;; When :all-pages-public? true, export repo would export all pages within that repo.
- ;; Regardless of whether you've set any page to public or not.
- ;; Example:
- ;; :publishing/all-pages-public? true
-
- ;; Specify default home page and sidebar status for Logseq
- ;; If not specified, Logseq default opens journals page on startup
- ;; value for `:page` is name of page
- ;; Possible options for `:sidebar` are
- ;; 1. `"Contents"` to open up `Contents` in sidebar by default
- ;; 2. `page name` to open up some page in sidebar
- ;; 3. Or multiple pages in an array ["Contents" "Page A" "Page B"]
- ;; If `:sidebar` is not set, sidebar will be hidden
- ;; Example:
- ;; 1. Setup page "Changelog" as home page and "Contents" in sidebar
+ ;; Default value: tab
```

Comment:
```suggestion
 ;; Default value: :tab
```
I think we should include the `:` since it's required (as seen 5 lines below this)